lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -435,20 +503,46 @@ sub get_test_results {
                     $res->{message} = 'UNKNOWN POLICY FORMAT';
                 }
             }
-            elsif ( $res->{message} =~ /config\.json/ ) {
-                my ( $config ) = ( $res->{message} =~ /\s(\/.*)$/ );
-                if ( $config ) {
-                    my $config_description = 'DEFAULT CONFIGURATION';
-                    $config_description = 'SOME OTHER CONFIGURATION' if ( $config =~ /some\/other\/configuration\/path/ );
-                    $res->{message} =~ s/$config/$config_description/;
+
+            $res->{module} = $test_res->{module};
+            $res->{message} = $translator->translate_tag( $test_res, $browser_lang ) . "\n";
+            $res->{message} =~ s/,/, /isg;
+            $res->{message} =~ s/;/; /isg;
+            $res->{level} = $test_res->{level};
+
+            if ( $test_res->{module} eq 'SYSTEM' ) {
+                if ( $res->{message} =~ /policy\.json/ ) {
+                    my ( $policy ) = ( $res->{message} =~ /\s(\/.*)$/ );
+                    if ( $policy ) {
+                        my $policy_description = 'DEFAULT POLICY';
+                        $policy_description = 'SOME OTHER POLICY' if ( $policy =~ /some\/other\/policy\/path/ );
+                        $res->{message} =~ s/$policy/$policy_description/;
+                    }
+                    else {
+                        $res->{message} = 'UNKNOWN POLICY FORMAT';
+                    }
```

Comment:
This is (almost) repeated from above.

In fact now that I look at it there's lots of code that looks like it could be thrown out here. We don't have config and policy anymore, so we shouldn't need special handling of messages related to those either.